content/guides/references/experiments.md

```diff
@@ -25,14 +25,15 @@ configuration to Cypress.
 | ---------------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `experimentalFetchPolyfill`        | `false` | Automatically replaces `window.fetch` with a polyfill that Cypress can spy on and stub. Note: `experimentalFetchPolyfill` has been deprecated in Cypress 6.0.0 and will be removed in a future release. Consider using [cy.intercept()](/api/commands/intercept) to intercept `fetch` requests instead. |
 | `experimentalInteractiveRunEvents` | `false` | Allows listening to the [`before:run`](/api/plugins/before-run-api), [`after:run`](/api/plugins/after-run-api), [`before:spec`](/api/plugins/before-spec-api), and [`after:spec`](/api/plugins/after-spec-api) events in the plugins file during interactive mode.                                      |
-| `experimentalSessionSupport`       | `false` | Enable use of [`cy.session()`](/api/commands/session) and new behavior to handle caching and restoring cookies, localStorage, and sessionStorage.                                                                                                                                                       |
+| `experimentalSessionAndOrigin`     | `false` | Enable use of [`cy.session()`](/api/commands/session) and [`cy.origin()`](/api/commands/session), and new behavior to handle caching and restoring cookies, localStorage, and sessionStorage.                                                                                                           |
```

Comment:
This is the statement we use in the code:
"Enables cross-origin and improved session support, including the `cy.origin` and `cy.session` commands."